job/common/src/main/java/alluxio/job/plan/replicate/MoveConfig.java

```diff
@@ -72,7 +72,7 @@ public String getName() {
 
   @Override
   public Collection<String> affectedPaths() {
-    if (StringUtils.isEmpty(mPath)) {
+    if (!StringUtils.isEmpty(mPath)) {
```

Comment:
were we not moving anything all this time?

Reply:
we weren't displaying them in the affectedPaths